### DISCUSSION TITLE
Can I have separate two tabs, one top and one bottom?

### DISCUSSION THREAD

**Discussion by saudiqbal** [Eugeny/tabby#9489] [Q&A]
I am not talking about split tabs, I need to see two different servers on the same window. I am not seeing any option to do this. Is this supported in Tabby?

**Comment by Eugeny** [comment]
Yes, you can drag other tabs into your active tab and dock them as panes.

**Reply by Eugeny** [reply]
Or split as usual, then right click -> "switch profile"

**Reply by saudiqbal** [reply]
Dragging is not working, it keeps moving the other tab.

**Reply by Eugeny** [reply]
https://github.com/Eugeny/tabby/assets/161476/be4656de-531e-43ee-bdbc-21df4ae7006a

**Reply by saudiqbal** [reply]
I am not seeing this happening with Kubuntu, could be just a Windows function

**Reply by saudiqbal** [reply]
Ok it worked, Found it

**Reply by Eugeny** [reply]
It's not platform dependent. Can you make a screencast of you trying this?

**Reply by saudiqbal** [reply]
One last question, how can I undo it, I want the split pane back into its own tab?

**Reply by Eugeny** [reply]
Hold the "rearrange panes" hotkey (default varies by platform) and drag handles will appear on each pane.

**Comment by saudiqbal** [comment]
One more thing please add a dedicated space for Options and close button, It hides the tab completely.

https://github.com/Eugeny/tabby/issues/8542

**Reply by Eugeny** [reply]
There's also an "explode-tab" hotkey to immediately turn all panes into separate tabs

**Reply by saudiqbal** [reply]
I posted a screenshot which shows the close and option buttons covers the whole tab completely.
